ta/pkcs11/src/object.c

```diff
@@ -999,9 +999,14 @@ enum pkcs11_rc entry_set_attribute_value(struct pkcs11_client *client,
 	if (rc)
 		goto out;
 
+	if (get_bool(obj->attributes, PKCS11_CKA_TOKEN)) {
+		rc = update_persistent_object_attributes(obj);
+		if (rc)
+			goto out;
+	}
+
 	DMSG("PKCS11 session %"PRIu32": set attributes %#"PRIx32,
 	     session->handle, object_handle);
-
```

Comment:
removing this line is not needed. I prefer you discard this suppression so that commit only does what it pretends to: add a missing piece.